network_modeling/configs/templates/1.0/aruba/common/cdu-to-spine.lag.j2

```diff
@@ -7,8 +7,8 @@ interface lag 255 multi-chassis
     no shutdown
     description cdu_to_spines_lag
     no routing
-    vlan trunk native {{ variables.NATIVE_VLAN }}
-    vlan trunk allowed {{ variables.LEAF_BMC_VLANS }}
+    vlan trunk native 1
```

Comment:
More work required to make these dynamic?  Why this change?

Reply:
1.0 templates, 1.2 templates are all dynamic.. the CMM vlan breaks 1.0.